content/doc/book/system-administration/security.adoc

```diff
@@ -74,16 +74,49 @@ The following topics discuss other security features that are on by default. You
 One may accidentally set up a security realm / authorization in such a way that
 you may no longer be able to reconfigure Jenkins.
 
-When this happens, you can fix this by the following steps:
+When this happens, the steps to deactivate the Authorization realm depend on the way you manage your configuration.
+
+[WARNING]
+====
+After following the next sections, when Jenkins comes back, it will be in an unsecured mode where everyone gets full
+access to the system.
+====
+
+=== Using JCasc
+
+Locate your configuration as code file, default is `$JENKINS_HOME/jenkins.yaml` but it can be located in a number of places.
+Review https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/README.md[the Jenkins Configuration as Code plugin for details].
+
+Once you have located the file:
+
+. Modify the `authorizationStrategy` section, so that it configures an unsecured realm:
++
+[source, yaml]
+----
+jenkins:
+  authorizationStrategy:
+    unsecured
```

Comment:
shouldn't it be this:
```suggestion
  authorizationStrategy: unsecured
```

Reply:
I validated the one I proposed. I guess it's just a question of style then :)

Reply:
ah didn't know that was valid yaml if it works then i guess it's okay

Reply:
You made me doubt, so I double checked. It's indeed a valid YAML but my linter prefers your form so let's take yours :)